src/region/RegionAnnotations.tsx

```diff
@@ -110,7 +110,7 @@ export default class RegionAnnotations extends React.PureComponent<Props, State>
                 {/* Layer 3a: Staged (unsaved) annotation target, if any */}
                 {canCreate && staged && (
                     <div className="ba-RegionAnnotations-target">
-                        <RegionRect ref={this.setRectRef} isActive shape={staged.shape} />
+                        <RegionRect ref={this.setRectRef} isActive shape={(staged as CreatorRegion).shape} />
```

Comment:
We should really try to avoid explicit casting like this, since it basically disables the type check entirely. Can we use a type guard or something, instead?